script/foreman-rake

```diff
@@ -10,7 +10,9 @@ die() {
 cd ~foreman || die "Cannot change to foreman home directory"
 [ -f Gemfile ] && BUNDLER_CMD="bundle exec"
 CMD="$BUNDLER_CMD $RAKE_CMD"
-if [ $USERNAME = foreman ]; then
+if [ $# -eq 0 ]; then
+  foreman-rake -h; exit 0;
```

Comment:
how about using `foreman-rake -D -A` instead: this prints all available tasks (not that all would be needed for us. Maybe the foreman-rake should have it's own help just with relevant commands, but that's probably for different issue.